salt/kube-apiserver/init.sls

```diff
@@ -12,6 +12,13 @@ include:
          cn = grains['nodename'],
          o = pillar['certificate_information']['subject_properties']['O']) }}
 
```

Comment:
You could add here and generate certs only if doesn't exist yet or wasn't provided by user:
```
{% if not ( pillar['ssl']['kube_apiserver_aggregator_crt'] or pillar['ssl']['kube_apiserver_aggregator_key'] ) %}
```